tiledb/sm/filesystem/posix.cc

```diff
@@ -47,6 +47,7 @@
 #include <unistd.h>
 
 #include <algorithm>
+#include <format>
```

Comment:
We have to use `fmt`; not all compilers we build on support `format`.

Reply:
Derp I forgot.

Thanks!

Reply:
(So, I'm embarrassed to have forgotten here. ;) )